qCC/ccGraphicalTransformationTool.cpp

```diff
@@ -218,7 +296,7 @@ void ccGraphicalTransformationTool::advTranslateRefChanged(int index)
 	}
 	int id = translateComboBox->itemData(index).toInt();
 	int selectedObjectIndex = -1;
-	for (int i; i < m_planesAndLineSegments.size(); i++)
+	for (int i = 0; i < m_planesAndLineSegments.size(); i++)
```

Comment:
I believe it should be size_t instead of int (if m_planesAndLineSegments is a vector, or a ccHObject::Container = vector)

Reply:
Same comment for other places

Reply:
Sure, ill change that